app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -215,6 +215,17 @@
 
 .gem-c-image-card__youtube-video-embed iframe {
   z-index: 1;
+  transform: scale(0.7);
+  width: calc(100% + 42px);
+}
+
+.gem-c-image-card__youtube-video-embed.gem-c-govspeak__youtube-video iframe {
+  top: -8px;
+  left: -16px;
```

Comment:
This is obviously a complex solution - would it be worth adding a comment to explain these numbers in particular?

Reply:
have added comments to the all the rules pertaining the youtube embed

Reply:
Well explained, good stuff.